src/regexbench.cpp

```diff
@@ -225,7 +228,7 @@ Arguments parse_options(int argc, const char *argv[]) {
   po::options_description optargs("Options");
   optargs.add_options()("help,h", "Print usage information.");
   optargs.add_options()(
-      "engine,e", po::value<std::string>(&engine)->default_value("hyperscan"),
+      "engine,e", po::value<std::string>(&engine)->default_value("rematch"),
```

Comment:
Sorry, I made that change thinking that this we wanted to showcase our own product.  I will change it back.